api/core/tools/provider/builtin/gitlab/tools/gitlab_mergerequests.yaml

```diff
@@ -8,7 +8,7 @@ description:
   human:
     en_US: A tool for query GitLab merge requests, Input should be a exists reposity or branch.
```

Comment:
Fix a similar typo in this line.

Reply:
thanks, it's done!
https://github.com/langgenius/dify/pull/11359/commits/794be971126dc3769ec325d4bb9c63bbe705aea9